dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -0,0 +1,408 @@
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Utility class used to map query results to POJO objects
+ * @author nollymar
+ */
+public class ConvertToPOJOUtil {
+
+    public static final DateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    private static final String SYSPUBLISH_DATE = "syspublish_date";
+    private static final String SYSEXPIRE_DATE = "sysexpire_date";
+    private static final String TITLE = "title";
+    private static final String MOD_DATE = "mod_date";
+    private static final String MOD_USER = "mod_user";
+    private static final String SORT_ORDER = "sort_order";
+    private static final String IDENTIFIER = "identifier";
+    private static final String SHOW_ON_MENU = "show_on_menu";
+    private static final String FRIENDLY_NAME = "friendly_name";
+    private static final String DRAWED = "drawed";
+    private static final String MAX_CONTENTLETS = "max_contentlets";
+    private static final String USE_DIV = "use_div";
+    private static final String STATICIFY = "staticify";
+    private static final String ADD_CONTAINER_LINKS = "add_container_links";
+    private static final String CONTAINERS_ADDED = "containers_added";
+    private static final String INODE = "inode";
+
+    private ConvertToPOJOUtil(){
+
+    }
+
+    /**
+     * Creates new instances of T given a List<Map> of DB results
+     * @param results
+     * @param classToUse
+     * @param <T>
+     * @return
+     * @throws Exception
+     */
+    public static<T> List<T> convertDotConnectMapToPOJO(List<Map<String,String>> results, final Class classToUse)
+            throws ParseException, IllegalAccessException, InvocationTargetException, InstantiationException, NoSuchMethodException, NoSuchFieldException {
+
+        List<T> ret = null;
+
+        if(results == null || results.isEmpty()){
+            return Collections.emptyList();
+        }
+
+        if (Folder.class.equals(classToUse)){
+            ret = (List<T>) convertDotConnectMapToFolder(results);
+        }
+
+        if (Container.class.equals(classToUse)){
+            ret = (List<T>) convertDotConnectMapToContainer(results);
+        }
+
+        if (Link.class.equals(classToUse)){
+            ret = (List<T>) convertDotConnectMapToLink(results);
+        }
+
+        if (Identifier.class.equals(classToUse)){
+            ret = (List<T>) convertDotConnectMapToIdentifier(results);
+        }
+
+        if (Template.class.equals(classToUse)){
+            ret = (List<T>) convertDotConnectMapToTemplate(results);
+        }
+
+        if (ret != null){
+            return ret;
+        }
+
+        return getMapFields(results, classToUse);
+    }
+
+    @NotNull
+    private static <T> List<T> getMapFields(List<Map<String, String>> results, Class classToUse)
+            throws NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException, NoSuchFieldException, ParseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 155 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)